cmd/juju/model/exportbundle.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package model
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+
+	"github.com/juju/juju/api/bundle"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/cmd/output"
+)
+
+// NewExportBundleCommand returns a fully constructed export bundle command.
+func NewExportBundleCommand() cmd.Command {
+	cmd := &exportBundleCommand{}
+	cmd.newAPIFunc = func() (ExportBundleAPI, error) {
+		return cmd.getAPI()
+	}
+	return modelcmd.Wrap(cmd)
+}
+
+type exportBundleCommand struct {
+	modelcmd.ModelCommandBase
+	out        cmd.Output
+	newAPIFunc func() (ExportBundleAPI, error)
+	Filename   string
+}
+
+const exportBundleHelpDoc = `
+Exports the current model configuration as bundle.
+
+--filename specifies the file where bundle is written to. Its optional.
```

Comment:
Maybe @pmatulis can help phrasing this? :D

Reply:
Thanks anastasiamac. @pmatulis Could you please help us in phrasing this new export-bundle command doc ?

Reply:
Conceptually, I see the workflow as a) creating a bundle and then b) exporting it. Therefore:

```
Exports a bundle.
...
If --filename is not used, the configuration is printed to stdout.
...
 --filename specifies an output file.
...
```

That being said, is it actually useful to send output to STDOUT? I'm wondering whether we should not simply enforce an output file.

Reply:
The user  can also manually redirect the output to a file instead of STDOUT which is same as providing the --fielname option.

Reply:
Thank you @pmatulis